framework/src/multiapps/MultiApp.C

```diff
@@ -776,6 +781,64 @@ MultiApp::getCommandLineArgsParamHelper(unsigned int local_app)
     return _cli_args[local_app + _first_local_app];
 }
 
+LocalRankConfig
+rankConfig(unsigned int rank,
+           unsigned int nprocs,
+           unsigned int napps,
+           unsigned int min_app_procs,
+           unsigned int max_app_procs)
+{
+  if (min_app_procs > nprocs)
+    mooseError("minimum number of procs per app is higher than the available number of procs");
+  else if (min_app_procs > max_app_procs)
```

Comment:
Should this just be an "if"

Reply:
Doesn't matter - since if the other condition fired - the app terminated with an error already.